DOCS-1781: Handle planning for the zero pose (#2589)
viamrobotics/docs: docs/mobility/motion/_index.md

### MoveOnMap

Move a [base](/components/base/) component to a destination [`Pose`](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose) on a {{< glossary_tooltip term_id="slam" text="SLAM" >}} map.

`MoveOnMap()` is non blocking, meaning the motion service will move the component to the destination [`Pose`](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose) after `MoveOnMap()` returns.

> Each successful `MoveOnMap()` call returns a unique `ExecutionID` which you can use to identify all plans generated during the `MoveOnMap()` call.

Reviewer: This also applies to `MoveOnGlobe()`. May you please update that as well?

Author: done, thanks

{{< alert title="Info" color="info" >}}

> If you specify a goal pose or orientation and the robot's current position is already at the goal post or orientation, `MoveonMap` returns an error.

Reviewer: small nit: I would say:
```
If you specify a goal pose and the robot's current position is already within the set `PlanDeviationM`, then `MoveonMap` returns an error.
```

i.e. if we are within PlanDeviationMM of the goal pose then we do not need to start.

> If you specify a goal pose or orientation and the robot's current position is already at the goal post or orientation, `MoveonMap` returns an error.

Author: ```suggestion
If you specify a goal pose and the robot's current position is already within the set `PlanDeviationM`, then `MoveonMap` returns an error.
```

Move a [base](/components/base/) component to a destination GPS point, represented in geographic notation _(latitude, longitude)_.
Use a [movement sensor](/components/movement-sensor/) to check the location of the machine.

`MoveOnGlobe()` is non blocking, meaning the motion service will move the component to the destination GPS point after `MoveOnGlobe()` returns.

> Each successful `MoveOnGlobe()` call retuns a unique `ExecutionID` which you can use to identify all plans generated during the `MoveOnGlobe()`.

Author: ```suggestion
Each successful `MoveOnGlobe()` call returns a unique `ExecutionID` which you can use to identify all plans generated during the `MoveOnGlobe()`.
```

{{< alert title="Info" color="info" >}}

> If you specify a goal pose or orientation and the robot's current position is already at the goal post or orientation, `MoveonMap` returns an error.

Author: ```suggestion
If you specify a goal pose or orientation and the robot's current position is already within the set `PlanDeviationM` or orientation, `MoveOnGlobe` returns an error.
```

> If you specify a goal pose or orientation and the robot's current position is already within the set `PlanDeviationM` or orientation, `MoveOnGlobe` returns an error.

Reviewer: can you please change this to be
```
If you specify a goal pose and the robot's current position is already within the set `PlanDeviationM`,  `MoveOnGlobe` returns an error.
```

Reviewer: @npentrel this should not include `"or orientation"`

> If you specify a goal pose or orientation and the robot's current position is already within the set `PlanDeviationM` or orientation, `MoveOnGlobe` returns an error.

Author: ```suggestion
If you specify a goal pose and the robot's current position is already within the set `PlanDeviationM`, `MoveOnGlobe` returns an error.
```
